src/pudl/helpers.py

```diff
@@ -1589,7 +1590,10 @@ def post_process(self, value: str) -> str:
         """
         if value is None:
             try:
-                return os.environ[self.env_var]
+                value = os.environ.get(self.env_var)
```

Comment:
using `os.environ.get(self.env_var, get_defaults()[self.env_var])` will evaluate `get_defaults`, which causes the CI to fail when no `.pudl.yaml` file exists, so I've separated these.

Reply:
Easier to read, too!